Dockerfile

```diff
@@ -1,10 +1,11 @@
 # Build Agora from source
 FROM bosagora/agora-builder:latest AS Builder
 ARG DUB_OPTIONS
+ARG DISABLE_TALOS
 ARG AGORA_VERSION="HEAD"
 ADD . /root/agora/
 WORKDIR /root/agora/talos/
-RUN npm ci && npm run build
+RUN if [ -z ${DISABLE_TALOS+x} ]; then npm ci && npm run build; fi
```

Comment:
See: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash
So `DISABLE_TALOS="false"` will still disable Talos. I'm not too bothered about this though :)